README.md

```diff
@@ -12,7 +12,7 @@ Where `<index name>` is an assumed existing index within the given technology, a
 
 The `<technology>` in the triple references the indexing technology, currently only suppored by `elasticsearch`.  This string is used to dynamically build the policy invocations when the indexing policy is triggered in order to delegate the operations to the appropriate rule engine plugin.
 
-The attribute is configurable within the `plugin_specific_configuration` for the indexing rule engine plugin.
+The attribute is configurable within the plugin_specific_configuration for the indexing rule engine plugin.
```

Comment:
Personally, I find the formatting more useful

Reply:
agreed